packages/components/src/Notification/GlobalNotification/_docs/GlobalNotification.stickersheet.stories.tsx

```diff
@@ -94,6 +135,15 @@ const StickerSheetTemplate: StickerSheetStory = {
           ))}
         </StickerSheet.Body>
       </StickerSheet>
+      <StickerSheet heading="Type (deprecated)" isReversed={isReversed}>
+        <StickerSheet.Body>
+          {TYPE_PROPS.map(({ title, props }) => (
+            <StickerSheet.Row key={title} rowTitle={title}>
+              <GlobalNotification {...props} />
```

Comment:
Could save you some lines? :)
```suggestion
              <GlobalNotification {...DEFAULT_PROPS} {...props} />
```